data/assets/examples/renderable/renderablesphereimageonline/sphereimageonline_fisheye.asset

```diff
@@ -0,0 +1,27 @@
+-- Fisheye Mapping
+-- This example shows a sphere that is covered with an image which is retrieved from an
+-- online URL and mapped to the sphere using Angular Fisheye projection. The image will
+-- cover half of the sphere.
+
+local Node = {
+  Identifier = "RenderableSphereImageOnline_Example_FisheyeMapping",
+  Renderable = {
+    Type = "RenderableSphereImageOnline",
+    URL = "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg",
+    TextureProjection = "Angular Fisheye",
+    -- Set orientation to also render the inside of the sphere
```

Comment:
Should we also add here that it is the correct view for a fisheye/fulldome image?